pkgs/development/python-modules/jinja2/2.nix

```diff
@@ -24,7 +24,7 @@ buildPythonPackage rec {
   doCheck = !stdenv.is32bit || isPy3k;
 
   checkPhase = ''
-    pytest -v tests
+    pytest -v tests -W ignore::ResourceWarning -W ignore::DeprecationWarning
```

Comment:
This change should have gotten its own commit. Instead it was stuffed into `tribler:  7.4.4 -> 7.10.0`.

Reply:
I agree. The original PR was complicated and took months. These contributions came from another contributor; @mkg20001 do you remember what was it about?

Reply:
Tests failing without it, as they were failing due to warnings being generated

Reply:
As for the squash everything was already mixed up. Initially I had a patch for the other commit, later squashed both together and it ended up like that,